cfgov/jinja2/v1/_includes/organisms/mega-menu.html

```diff
@@ -1,61 +1,146 @@
+{% import '_vars-mega-menu.html' as vars with context %}
+{% set base_class = 'o-mega-menu' %}
 
 {# ==========================================================================
 
-   sub_nav()
+   _classes()
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/8fb4d26598699a1cf1454bc9a26ea48bebbb98b0